account_financial_report/README.rst

```diff
@@ -42,6 +42,8 @@ currency used in account move lines is properly shown.
 In case that in an account has not been configured a second currency foreign
 currency balances are not available.
 
+Added `Hide Partners without Moves` filter: This filter hides the moves where they don't have `partner_id`
```

Comment:
@YogeshMahera-SerpentCS I think there is misunderstanding on the requirement. The requirement is to not show partners with no moves, which is different to hiding moves with no partner.

Reply:
![Peek 2020-08-05 22-34](https://user-images.githubusercontent.com/24264979/89443421-e2851100-d76d-11ea-8be2-5514244cf6b4.gif)

Hi, @yostashiro I have added sample GIF and Can you please check above sample GIF?

Reply:
@YogeshMahera-SerpentCS I can confirm that the requirement has been misunderstood.